src/uct/ib/mlx5/dv/ib_mlx5_ifc.h

```diff
@@ -85,9 +88,10 @@ enum {
 };
 
 enum {
-    UCT_IB_MLX5_CAP_GENERAL = 0,
-    UCT_IB_MLX5_CAP_ODP     = 2,
-    UCT_IB_MLX5_CAP_ATOMIC  = 3,
+    UCT_IB_MLX5_CAP_GENERAL   = 0,
+    UCT_IB_MLX5_CAP_ODP       = 2,
+    UCT_IB_MLX5_CAP_ATOMIC    = 3,
+    UCT_IB_MLX5_CAP_2_GENERAL = 0x20,
```

Comment:
please align dec/hex in scope of the enum

Reply:
@evgeny-leksikov Fixed.

Reply:
pls rename across the PR cap_2/CAP_2 -> cap2/CAP2 because often this_2_that means "this to that"

Reply:
@evgeny-leksikov The _2 of cap_2 is defined in PRM. When I did code for FW verification, a general requirement is follow the naming with PRM, then later we can easily seach the corresponding defines from PRM according to the names of variables. But if this againsts the code format rule, it's fine for me to change _2 to 2. Just want to double check. Thanks!

Reply:
@yosefe WDYT?

Reply:
agree with @leibin2014  - whatever is in ib_mlx5_ifc.h should match prm, literally, even if naming is not good